### DISCUSSION TITLE
Changing keyboard layout to us-acentos

### DISCUSSION THREAD

**Discussion by sitolam** [hyprwm/Hyprland#3750] [Q&A] [answered]
I want to change my keyboard layout from `us` to `us-acentos` I tried some things but  hyprland didn't want to boot after these changes. So how can I do it?
This is my current `hyprland.conf` input section: 
```bash
input {
    kb_layout = us
    kb_variant = 
    kb_model =
    kb_options =
    kb_rules =
    follow_mouse = 1

    touchpad {
        natural_scroll = no
    }

    sensitivity = 0 # -1.0 - 1.0, 0 means no modification.
    force_no_accel = 1
}
```

**Comment by varris1** [comment]
us-acentos as in US international? If that's the case, try

kb_layout = us
kb_variant = altgr-intl

**Reply by sitolam** [reply]
Yes, but I also want to be able to typ for example: é. So I want deadkeys.

**Reply by varris1** [reply, marked as answer]
then that should be (tested it myself):
kb_variant intl

**Reply by sitolam** [reply]
Yep that worked setting `kb_variant = intl` worked.

**Reply by vsantos1711** [reply]
hello, can u use the `' "` without pressing the `alt gr` ?  @Sitolam

**Reply by sitolam** [reply]
> hello, can u use the `' "` without pressing the `alt gr` ?  @Sitolam 

Yes, but with dead keys you need to first click the button of `' "` and then space.

**Reply by vsantos1711** [reply]
Can you pass me your config file, please?

**Reply by sitolam** [reply]
```bash
input {
    kb_layout = us
    kb_variant = intl
    kb_model =
    kb_options =
    kb_rules =
    follow_mouse = 1

    touchpad {
        natural_scroll = no
    }

    sensitivity = 0 # -1.0 - 1.0, 0 means no modification.
    force_no_accel = 1
}
```

**Reply by douglara** [reply]
> ```shell
> input {
>     kb_layout = us
>     kb_variant = intl
>     kb_model =
>     kb_options =
>     kb_rules =
>     follow_mouse = 1
> 
>     touchpad {
>         natural_scroll = no
>     }
> 
>     sensitivity = 0 # -1.0 - 1.0, 0 means no modification.
>     force_no_accel = 1
> }
> ```

Thanks friend!
